Translations/translation_RO.json

```diff
@@ -0,0 +1,214 @@
+    },
+    "SettingsReset": {
+      "text2": ["Setări", "din fabrică"],
+      "desc": "Reveniti la setările din fabrică"
+    },
+    "VoltageCalibration": {
+      "text2": ["Calibrare tens.", "de intrare?"],
+      "desc": "Porniti calibrarea VIN (apăsati lung pentru a iesi)"
+    },
+    "AdvancedSoldering": {
+      "text2": ["Detalii", "ecran lipire"],
+      "desc": "Afisati informatii detaliate într-un font mai mic pe ecranul de lipire"
+    },
+    "ScrollingSpeed": {
+      "text2": ["Viteză", "derulare"],
+      "desc": "Viteză derulare text cu informatii la (S=lent | F=rapid)"
+    },
+    "QCMaxVoltage": {
+      "text2": ["QC", "voltaj"],
+      "desc": "Tensiunea maximă QC dorită pentru care negociază letconul"
+    },
+    "PDNegTimeout": {
+      "text2": ["PD", "timeout"],
+      "desc": "PD negotiation timeout in 100ms steps for compatibility with some QC chargers"
+    },
+    "PowerLimit": {
+      "text2": ["Putere", "limită"],
+      "desc": "Puterea maximă pe care letconul o poate folosi (W=watt)"
+    },
+    "ReverseButtonTempChange": {
+      "text2": ["Inversare", "+ - butoane"],
+      "desc": "Inversarea butoanelor de reglare a temperaturii"
+    },
+    "TempChangeShortStep": {
+      "text2": ["Schimbare temp.", "apăsare scută"],
+      "desc": "Schimbarea temperaturii la apăsarea scurtă a butonului"
+    },
+    "TempChangeLongStep": {
+      "text2": ["Schimbare temp.", "apăsare lungă"],
+      "desc": "Schimbarea temperaturii la apăsarea lungă a butonului"
+    },
+    "PowerPulsePower": {
+      "text2": ["Putere", "puls"],
+      "desc": "Intensitatea puterii la păstrati-pulsul-treaz (watt)"
+    },
+    "HallEffSensitivity": {
+      "text2": ["Sensibilitate", "senzor Hall"],
+      "desc": "Sensibilitate senzor cu efect Hall pentru a detecta repausul (O=oprit | L=scăzut | M=mediu | H=înalt)"
+    },
+    "LockingMode": {
+      "text2": ["Blocare", "butoane"],
+      "desc": "Când lipiti, apăsati lung ambele butoane, pentru a le bloca (D=dezactivare | B=numai \"modul boost\" | F=blocare completă)"
+    },
+    "MinVolCell": {
+      "text2": ["Voltaj", "minim"],
+      "desc": "Tensiunea minimă admisă pe celulă (3S: 3 - 3.7V | 4-6S: 2.4 - 3.7V)"
+    },
+    "AnimLoop": {
+      "text2": ["Animatii", "buclă"],
+      "desc": "Animatii de pictograme în meniul principal"
+    },
+    "AnimSpeed": {
+      "text2": ["Animatii", "viteză"],
+      "desc": "Ritmul animatiilor pictogramei din meniu (O=oprit | S=încet | M=mediu | F=rapid)"
+    },
+    "PowerPulseWait": {
+      "text2": ["Întârziere", "puls putere"],
+      "desc": "Întârziere înainte de declansarea pulsului trezit (x 2.5s)"
+    },
+    "PowerPulseDuration": {
+      "text2": ["Durată", "puls putere"],
+      "desc": "Păstrati-pulsul-treaz durată (x 250ms)"
+    },
+    "LanguageSwitch": {
+      "text2": ["Limbă:", " RO     Română"],
+      "desc": "Current firmware language"
```

Comment:
@sandmanRO

Reply:
Limba curenta

Reply:
No problem.
As long as you like it, I will write it down for you.